### PR TITLE
[DE-241]: feat/add checkbox component

### PR DESCRIPTION
**Pull request by rafijv**
- Add checkbox component without radix library
- Add text/password input component
- Add textarea component

### CODE REVIEW

File: src/components/checkbox/Checkbox.tsx [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  checked = true,
+  indeterminate = false,
+  disabled = false,
+  onClick,
+  required,
+}: CheckboxComponentProps): JSX.Element => {
+  return (
+    <>
+      <button
+        onClick={(e) => {
```
Comment (xabg2) [inline src/components/checkbox/Checkbox.tsx:23]: Maybe you can pass onClick here directly and define it in the props as `onClick: (e: unknown) => void;`. This way, we modify the onClick outside the component and not add logic inside the components.

File: src/components/input/Input.tsx [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+interface InputComponentProps {
+  type?: 'text' | 'password';
+  size?: 'lg' | 'md';
+  disabled?: boolean;
+  accentColor?: 'red' | 'orange' | 'green';
+  placeholder?: string;
+}
+
+const Input = ({
```
Comment (xabg2) [inline src/components/input/Input.tsx:12]: How can we obtain the written text here?

File: src/components/textArea/TextArea.tsx [hunk 3]
```diff
@@ -0,0 +1,25 @@
+const TextArea = ({
```
Comment (xabg2) [inline src/components/textArea/TextArea.tsx:1]: Same as Input component.

File: src/stories/components/checkbox/Checkbox.stories.tsx [hunk 4]
```diff
@@ -0,0 +1,46 @@
+import type { Meta, StoryObj } from '@storybook/react';
+import Checkbox from '../../../components/checkbox/Checkbox';
+
+const meta: Meta<typeof Checkbox> = {
```
Comment (xabg2) [inline src/stories/components/checkbox/Checkbox.stories.tsx:4]: nit: if you want, you can make the component interactive, so you can click in the component in the story and see what happens. You an see [the radio button story](https://github.com/internxt/ui/blob/master/src/stories/components/radio-button/RadioButton.stories.ts#L6) to get an example.

File: src/components/checkbox/Checkbox.tsx (hunk 1 above)
Comment (rafijv) [reply src/components/checkbox/Checkbox.tsx]: Done

File: src/stories/components/checkbox/Checkbox.stories.tsx (hunk 4 above)
Comment (rafijv) [reply src/stories/components/checkbox/Checkbox.stories.tsx:17]: Done

File: src/components/textArea/TextArea.tsx (hunk 3 above)
Comment (rafijv) [reply src/components/textArea/TextArea.tsx:10]: value and onChange added

File: src/components/input/Input.tsx (hunk 2 above)
Comment (rafijv) [reply src/components/input/Input.tsx:15]: value and onChange added

File: src/components/checkbox/Checkbox.tsx [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+  indeterminate = false,
+  disabled = false,
+  onClick,
+  required,
+}: CheckboxComponentProps): JSX.Element => {
+  return (
+    <>
+      <button
+        onClick={onClick}
+        onKeyDown={() => {}}
```
Comment (xabg2) [inline src/components/checkbox/Checkbox.tsx:24]: Should we have a function like onClick for this? Just in case

Comment (rafijv) [reply src/components/checkbox/Checkbox.tsx]: Done

**Review by xabg2 — APPROVED** [review]

**Pull request merged**